deploy/ansible/config/vault_password

```diff
@@ -0,0 +1 @@
+|$<Uns3X]<.r(`fRy@g7XErTW:QE\$@w
```

Comment:
???

Reply:
Strange, it is gitignored. Anyway we don't need it since we got rid of encrypted vars